model/trace/gen-ai.yaml

```diff
@@ -1,21 +1,16 @@
 groups:
-  - id: gen_ai.request
+  - id: trace.gen_ai.client
     type: span
     brief: >
-      A request to an LLM is modeled as a span in a trace. The span name should be a low cardinality value representing the request made to an LLM, like the name of the API endpoint being called.
+      Describes GenAI operation span.
     attributes:
       - ref: gen_ai.system
         requirement_level: required
-        note: >
-          If not using a vendor-supplied model, provide a custom friendly name, such as a name of the company or project.
-          If the instrumetnation reports any attributes specific to a custom model, the value provided in the `gen_ai.system` SHOULD match the custom attribute namespace segment.
-          For example, if `gen_ai.system` is set to `the_best_llm`, custom attributes should be added in the `gen_ai.the_best_llm.*` namespace.
```

Comment:
removed since we don't have any system-specific attributes - we can add similar section back once we have any